### PR TITLE
Plug travis with tox

### PR DESCRIPTION
**Pull request by ametaireau**
It seems better to use tox for that.

### CODE REVIEW

**Comment by ametaireau** [comment]
@Natim @leplatrem r?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,17 @@
 language: python
-python:
-    - "2.6"
-    - "2.7"
-    - "3.3"
-install: pip install --use-mirrors .
-script: python setup.py test
+python: 2.7
+sudo: false
+services: redis-server
```
Comment (Natim) [inline .travis.yml]: Do you really need that?

Comment (ametaireau) [inline .travis.yml]: nope !)

**Comment by ametaireau** [comment]
updated

**Pull request merged**